schemas/performance-artifact.json

```diff
@@ -14,9 +14,10 @@
             "refbrow",
             "fenix",
             "safari",
-            "custom-car"
+            "custom-car",
+            "custom-car-m"
           ],
-          "maxLength": 10,
+          "maxLength": 12,
```

Comment:
This is limited in the database: https://github.com/mozilla/treeherder/blob/23a642030b126501e3a6000326a3fc9c0d384b8c/treeherder/perf/models.py#L57

Could we find a shorter name instead?

Reply:
ah I see. thanks @gmierz !
 how about `c-car-m`?

Reply:
`cust-car-m`? It's 10 😄
`cst-cat-m` it's 9.
`c-cat-m` seems to me a bit confusing.

Reply:
cstm-car-m? 10 :)